runs/comparisons/suprb_tuning.py

```diff
@@ -81,49 +83,91 @@ def suprb_ES_GA_space(trial: Trial, params: Bunch):
         #    params.rule_generation__delay = trial.suggest_int('rule_generation__delay', 10, 100)
 
-        #params.rule_generation__mutation = trial.suggest_categorical(
+        # params.rule_generation__mutation = trial.suggest_categorical(
         #    'mutation', ['Normal', 'HalfnormIncrease', 'UniformIncrease'])
-        #params.rule_generation__mutation = getattr(mutation, params.rule_generation__mutation)()
+        # params.rule_generation__mutation = getattr(mutation, params.rule_generation__mutation)()
 
         # GA
         params.solution_composition__selection = trial.suggest_categorical(
             'solution_composition__selection',
             ['RouletteWheel',
              'Tournament',
              'LinearRank', 'Random'])
-        params.solution_composition__selection = getattr(ga.selection, params.solution_composition__selection)()
+        params.solution_composition__selection = getattr(
+            ga.selection, params.solution_composition__selection)()
 
         if isinstance(params.solution_composition__selection, ga.selection.Tournament):
-            params.solution_composition__selection__k = trial.suggest_int('solution_composition__selection__k', 3, 10)
+            params.solution_composition__selection__k = trial.suggest_int(
+                'solution_composition__selection__k', 3, 10)
 
         params.solution_composition__crossover = trial.suggest_categorical('solution_composition__crossover',
                                                                            ['NPoint', 'Uniform'])
-        params.solution_composition__crossover = getattr(ga.crossover, params.solution_composition__crossover)()
+        params.solution_composition__crossover = getattr(
+            ga.crossover, params.solution_composition__crossover)()
 
         if isinstance(params.solution_composition__crossover, ga.crossover.NPoint):
-            params.solution_composition__crossover__n = trial.suggest_int('solution_composition__crossover__n', 1, 10)
+            params.solution_composition__crossover__n = trial.suggest_int(
+                'solution_composition__crossover__n', 1, 10)
 
         params.solution_composition__mutation__mutation_rate = trial.suggest_float(
             'solution_composition__mutation_rate', 0, 0.1)
 
-    experiment_name = f'ES Tuning & Experimentation {job_id} {problem}'
+        params.solution_composition__init__mixing__experience_weight = trial.suggest_float(
+            'solution_composition__init__mixing__experience_weight', 0, 2)
+        # params.solution_composition__init__mixing__experience_calculation__lower_bound = trial.suggest_float(
+        #     'solution_composition__init__mixing__experience_calculation__lower_bound', 0, 10)
+        params.solution_composition__init__mixing__experience_calculation__upper_bound = trial.suggest_float(
+            'solution_composition__init__mixing__experience_calculation__upper_bound', 20, 50)
```

Comment:
to make the script more readable a short reminder what this parameter does would be great. Here, tuning could be fine though

Reply:
Done